docker/docker-compose.yml

```diff
@@ -42,22 +56,22 @@ services:
     depends_on: #required while image doesnt support arm architecture
       - amd64_emulator
     profiles:
-      - raspberry
-      - server
+      - issuer
+      - holder
 
-  amd64_emulator:
+  amd64_emulator: #required by now because of problems in multiplatform builds of aca-py
     container_name: amd64
     image: tonistiigi/binfmt
     privileged: true
     command: "--install amd64"  
     profiles:
-      - raspberry
+      - holder
 
-  temp_collector:
-    container_name: temp_collector
-    image: elazzar/rpi-dht11-api-docker
-    privileged: true
-    ports: 
-      - 5000:5000
-    profiles:
-      - raspberry
+  # temp_collector:
```

Comment:
I think it's better to leave the code uncommented, so the real data from the sensors embedded in the raspberry will be collected by the container created